<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLOADING</class>
 <widget class="QDialog" name="CLOADING">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登陆系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tu.png</normaloff>:/tu.png</iconset>
  </property>
  <widget class="QLabel" name="Login_Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>96</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>96</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>96</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="QlinePass">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>186</y>
     <width>252</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="QlineCity">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>240</y>
     <width>252</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="QchkPassWord">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>298</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="QchkAutoLogin">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>298</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnRevert">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>338</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnLoad">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>338</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>165</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>165</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLineUser">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>132</y>
     <width>252</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>QlinePass</tabstop>
  <tabstop>QlineCity</tabstop>
  <tabstop>QpbtnLoad</tabstop>
  <tabstop>QpbtnRevert</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
